src/components/tools/Charts/DotAndWhisker/Tooltip.svelte

```diff
@@ -0,0 +1,72 @@
+<script>
+  export let evt = {};
+  export let offset = 35;
+  export let units;
+
+  function formatValue(d) {
+    return +Math.round(d * 100) / 100;
```

Comment:
you probably don't need the `+` sign here?

Reply:
removed in https://github.com/berkeley-gif/caladapt-website-2021/pull/211/commits/776de2beac4bfd7acf797bcc9d342752033e9924